src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,25 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    private final static int maxSise = 5;
```

Comment:
@joej0e Constants should be named WITH_CAPSLOCK_AND_UNDERSCRORE

Reply:
Travis said me that i can't 
Abbreviation in name 'MAX_SIZE' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]
[ERROR] /home/travis/build/mate-academy/jv-tree-set/src/main/java/core/basesyntax/TreeSetCharacters.java:27:23: Member name 'MAX_SIZE' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. [MemberName]

Reply:
OK